src/components/dom.jsx

```diff
@@ -70,7 +75,7 @@ const DOM = (props) => {
 			<body>
 				<div id='outlet' dangerouslySetInnerHTML={getInnerHTML(appMarkup)} />
 				<script dangerouslySetInnerHTML={getInnerHTML(`window.APP_RUNTIME=${JSON.stringify(APP_RUNTIME)};`)} />
-				<script src={`${assetPublicPath}${clientFilename}`} />
+				{scripts.map(url => <script src={url} />)}
```

Comment:
React throws a warning about inserting arrays of elements without `key` props - you can see the error generated by this line if you run the integration tests without `--silent`.

`.map` makes sense if you have a large or arbitrary length array, but since we really want to keep the number of `<script>` tags small and explicit, I recommend continuing to write out each `<script>` individually (do you expect more than 2-3 total?)

If you want to map, use

```jsx
{scripts.map((src, key) => <script key={key} src={src} />)}
```